lib/command/src/Obelisk/Command/Run.hs

```diff
@@ -202,21 +203,24 @@ warnDifferentLanguages _ = return ()
 runGhciRepl
   :: MonadObelisk m
   => FilePath -- ^ Path to .ghci
+  -> [String] -- ^ Extra ghci arguments
   -> m ()
-runGhciRepl dotGhci = inProjectShell "ghc" $ unwords $ "ghci" : ["-no-user-package-db", "-ghci-script", dotGhci]
+runGhciRepl dotGhci ghciArgs = inProjectShell "ghc" $ unwords $ "ghci" : ["-no-user-package-db", "-ghci-script", dotGhci] ++ ghciArgs
 
 -- | Run ghcid
 runGhcid
   :: MonadObelisk m
   => FilePath -- ^ Path to .ghci
+  -> [String] -- ^ Extra ghci arguments
   -> Maybe String -- ^ Optional command to run at every reload
   -> m ()
-runGhcid dotGhci mcmd = callCommand $ unwords $ "ghcid" : opts
+runGhcid dotGhci ghciArgs mcmd = callCommand $ unwords $ "ghcid" : opts
   where
     opts =
       [ "-W"
       --TODO: The decision of whether to use -fwarn-redundant-constraints should probably be made by the user
```

Comment:
Note that we could get rid of this `TODO` now if we moved `-fwarn-redundant-constraints` into the default list of ghci arguments and removed it from the built-in list.

Reply:
We also have the option of detecting "unspecified" and reverting to this as a default even for users who upgrade obelisk without modifying their `default.nix`.